python/nav/web/templates/ipdevinfo/frag-ipdevinfo-alert-box.html

```diff
@@ -1,3 +1,9 @@
-<div class="fixed-alert">
-  <div class="alert-box with-icon {{ alert_level }}">{{ alert_message }}<span class="close" hx-on:click="htmx.remove(this.closest('.alert-box'))">x</span></div>
+<div hx-swap-oob="afterend:#ipdevpoll-jobs">
+  <div class="alert-box with-icon {{ alert_level }}" data-nav-alert>
+      {{ alert_message }}
+      <span class="close">x</span>
+  </div>
 </div>
+<button class="table-button secondary refresh-button" disabled>
+    Refresh
+</button>
```

Comment:
This isn't any better than having the loading indicator spin forever... all the refresh buttons are now disabled, and there is no way to try again without reloading the whole page, which is not ideal.

The net effect of this is the same as before, but could I ask for a new PR that enables the refresh buttons after an error, so the user is free to try again?